conformance/tests/admin-network-policy-core-ingress-tcp-rules.go

```diff
@@ -56,10 +56,19 @@ var AdminNetworkPolicyIngressTCP = suite.ConformanceTest{
 				Name:      "harry-potter-0",
 			}, serverPod)
 			require.NoErrorf(t, err, "unable to fetch the server pod")
+
+			// ensure ingress is ALLOWED from dumbledore and DENIED from professor quirrell. Both pods are in the hogwarts-staff namespace
```

Comment:
nit: Maybe mention which rules (at index 7 and 8 in this particular case) causes the traffic to be allowed/dropped

Reply:
👍🏼  Remediated in the current ones, and will continue to add index markers as I work through the rest.